data/pools/transactionPool.go

```diff
@@ -50,21 +50,19 @@ type TransactionPool struct {
 	txPoolMaxSize          int
 
 	// pendingMu protects pendingTxGroups and pendingTxids
```

Comment:
add pendingVerifyParams to comment here and below

Reply:
I'll do this in a future change; good catch.